soh/soh/Enhancements/randomizer/3drando/playthrough.cpp

```diff
@@ -97,7 +97,6 @@ int Playthrough_Repeat(std::set<RandomizerCheck> excludedLocations, std::set<Ran
         ctx->GetSettings()->SetSeedString(std::to_string(rand() % 0xFFFFFFFF));
         repeatedSeed = boost::hash_32<std::string>{}(ctx->GetSettings()->GetSeedString());
         ctx->GetSettings()->SetSeed(repeatedSeed % 0xFFFFFFFF);
-        //CitraPrint("testing seed: " + std::to_string(Settings::seed));
```

Comment:
Probably worth keeping this as a SPDLOG print

Reply:
Done.